packages/core/src/xr/input/XRTrackedInputType.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Enumerates some devices that can be tracked.(including status, posture and other information)
+ */
+export enum XRTrackedInputType {
```

Comment:
Enum implies Type, you can omit it

Use `XRTrackedInputDevice` is better than `XRTrackedInputType` and more clearly!